bin/mv.js

```diff
@@ -141,31 +138,118 @@ function loadProfile(name) {
     return config.getProfile(name);
 }
 
-function moveCronJob(profile, name, target) {
+function moveCronJob(profile, name, target, options) {
     return coroutine(function*() {
-        var cronJobs = yield profile.listCronJobs();
-
-        for (let job of cronJobs) {
-            if (job.name === name) {
-                let token = yield profile.getWebtask({
-                    name: target.name
-                });
-                yield target.profile.createCronJob({
-                    name: target.name,
-                    container: target.container,
-                    token: token.token,
-                    schedule: job.schedule
-                });
-                yield profile.removeCronJob({
-                    name: name
-                });
-                break;
+        var job;
+
+        try {
+            job = yield profile.getCronJob({
+                name: name
+            });
+        } catch (err) {
+            if (err.statusCode === 404) {
+                return;
             }
+            throw err;
+        }
+
+        let webtask = yield profile.getWebtask({
+            name: target.name,
+            container: target.container
+        });
+
+        if (_.get(options, 'verify') && job.token !== options.verify) {
+            console.log(Chalk.bold('* Warning: failed to verify the cron job token (no match).'));
         }
+
+        yield target.profile.createCronJob({
+            name: target.name,
+            container: target.container,
+            token: webtask.token,
+            schedule: job.schedule
+        });
+
+        yield profile.removeCronJob({
+            name: name
+        });
+    })();
+}
+
+function copyStorage(webtask, target) {
+    return coroutine(function*() {
+        let targetWebtask = yield target.profile.getWebtask({
+            name: target.name,
+            container: target.container
+        });
+        let data = yield exportStorage(webtask);
```

Comment:
Yes, this is suboptimal, and support from the backend API is the way to go (IMHO).

Btw, In my tests, sometimes  `ctx.storage.get()` returns empty data, and in the next run the data *shows back* again.